Formula/rust.rb

```diff
@@ -95,5 +122,8 @@ def install
     system "#{bin}/cargo", "new", "hello_world", "--bin"
     assert_equal "Hello, world!",
                  (testpath/"hello_world").cd { `#{bin}/cargo run`.split("\n").last }
+    if build.with? "racer"
+      system bin/"racer", "complete", "std::io::B"
```

Comment:
can we actually match something here so that we're testing something other than a 0 exit code?